lib/components/molecules/StickyHeader/StickyHeader.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import React, { useState, useRef, useEffect } from 'react';
+import styled from 'styled-components';
+import { Props } from './types/index';
+import useScroll from '../../../hooks/useScroll';
+
+import styles, { Wrapper } from './StickyHeader.style';
+
+const StickyHeader = ({ children, className }: Props): Node => {
+  const [isSticky, setSticky] = useState(false);
+  const wrapperRef = useRef(null);
+
+  useEffect(() => {
+    window.addEventListener('scroll', () => useScroll(wrapperRef, setSticky));
+    return () => {
+      window.removeEventListener('scroll', useScroll);
+    };
+  });
+
+  return (
+    <Wrapper
+      aria-label="Sticky Header"
```

Comment:
Don't hard code any string in the component, take them through props please.

Reply:
aria-label wont work on non-interactive elements like Wrapper (originally a div tag)